### PR TITLE
Start JupyterLab with  'jupyter lab'

### PR DESCRIPTION
**Pull request by SylvainCorlay**
On the notebook repo side, we should create a base class for `NotebookApp`, `ServerApp` which does not hold anything notebook specific.

### CODE REVIEW

File: jupyterlab/labapp.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TODO: import base server app
```
Comment (jasongrout) [inline jupyterlab/labapp.py]: Needs the standard project copyright/license headers.

**Comment by jasongrout** [comment]
This looks great and tests great. If you can fix the one comment, I'll merge. Thanks!

**Comment by SylvainCorlay** [comment]
Done.

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**